python/GangaCore/Core/InternalServices/ShutdownManager.py

```diff
@@ -31,10 +31,16 @@ def register_exitfunc():
     registereing this  300 times is just bad...
     """
     if not register_exitfunc._has_registered:
-        atexit.register(_ganga_run_exitfuncs)
+        atexit.register(__ganga_run_exitfuncs)
         register_exitfunc._has_registered = True
 register_exitfunc._has_registered = False
 
+def __ganga_run_exitfuncs():
```

Comment:
So we have a function called `__ganga_run_exitfuncs` and one called `_ganga_run_exitfuncs`. It can't possibly be a good idea to have two functions with the same name apart from the number of underscores to begin with,

Reply:
Yeah I could probably name this better, I tried a wrapper but bad errors happened for me